Website/DesktopModules/Hotcakes/packages.config

```diff
@@ -6,11 +6,14 @@
   <package id="jQuery.UI.Combined" version="1.8.11" />
   <package id="jQuery.Validation" version="1.8.0" />
   <package id="log4net" version="1.2.10" targetFramework="net45" />
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.0.0" targetFramework="net472" />
   <package id="MiniProfiler" version="3.1.1.140" targetFramework="net40" />
   <package id="MiniProfiler.EF6" version="3.0.11" targetFramework="net40" />
   <package id="Modernizr" version="2.5.3" />
   <package id="MSBuild.Microsoft.VisualStudio.Web.targets" version="14.0.0.3" targetFramework="net451" />
   <package id="MSBuildTasks" version="1.5.0.235" targetFramework="net451" developmentDependency="true" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="4.5.3" targetFramework="net472" />
```

Comment:
This appears to be a new dependency. Was this required by Stripe or something else?

Reply:
In the last code review, I commented that the stripe solution was hosted in the Hotcakes project. In the current version, the Stripe.dll file comes from the references folder. I assume they compiled Stripe's solution to generate this solution with all its dependencies in a single dll file. With the NuGet approach, all dependencies must be added. This must be the reason why the new references appear. I have checked these packages on the nuget.org site and they are stripe.net dependencies